src/auth/auth.service.ts

```
@@ -1,26 +1,181 @@
+  ) {}
+
+  // 회원가입
+  async signup(dto: SignUpRequestDto) {
+    // 이메일 중복 확인
+    const emailExists = await this.prisma.user.findUnique({
+      where: { email: dto.email },
+    });
+
+    // 이메일이 이미 존재하면 예외 발생
+    if (!!emailExists) {
+      throw new ConflictException('이미 사용 중인 이메일입니다.');
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)